etl/helpers.py

```diff
@@ -118,3 +142,49 @@ def walden_dataset(self) -> WaldenDataset:
     @property
     def snapshot_dir(self) -> Path:
         return paths.SNAPSHOTS_DIR / self.namespace / self.version
+
+    @staticmethod
+    def _create_step_name(channel, namespace, version, short_name):
+        if channel in ["meadow", "garden", "grapher"]:
+            step_name = f"data://{channel}/{namespace}/{version}/{short_name}"
+        elif channel in ["walden", "snapshot"]:
+            step_name = f"{channel}://{namespace}/{version}/{short_name}"
+        else:
+            raise UnknownChannel
+
+        return step_name
+
+    @property
+    def step(self) -> str:
+        return self._create_step_name(
+            channel=self.channel, namespace=self.namespace, version=self.version, short_name=self.short_name
+        )
+
+    @property
+    def dependencies(self) -> List[str]:
+        return DAG[self.step]
+
+    def get_dataset_path(
+        self, channel: str, short_name: str, namespace: Optional[str] = None, version: Optional[Union[str, int]] = None
```

Comment:
Could `channel` be optional? I'd like to get dependency with as little as possible. E.g. `paths.deps["electricity_mix"]` that would give me `Dataset` instance (or Snapshot). I could then use a proper type in code. Something like

```python
def __get__(self) -> Union[Dataset, Snapshot]:
   ...

electricity_mix_ds: Dataset = paths.deps["electricity_mix"]
electricity_snapshot: Snapshot = paths.deps["electricity_mix"]
```